app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,8 @@
+module DeviseHelper
+  def devise_error_messages!
+    if resource.errors.full_messages.any?
+      flash.now[:error] = resource.errors.full_messages.join(' & ')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.